onnxruntime/core/providers/coreml/model/model.mm

```diff
@@ -334,11 +349,13 @@ - (Status)predict:(const std::unordered_map<std::string, OnnxTensorData>&)inputs
         switch (onnx_data_type) {
           case ONNX_NAMESPACE::TensorProto_DataType_FLOAT: {
             const auto output_data_byte_size = num_elements * sizeof(float);
+            ORT_RETURN_IF_NOT(coreml_buffer_size == output_data_byte_size, "CoreML output buffer size and expected output size differ");
```

Comment:
nit: 120 character line length limit

https://github.com/microsoft/onnxruntime/blob/ac4e72604605be524f70d1e92da81b70c5db984a/docs/Coding_Conventions_and_Standards.md?plain=1#L7-L8

Reply:
sorry I didn't see this wasn't automatic on purpose
https://github.com/microsoft/onnxruntime/blob/ac4e72604605be524f70d1e92da81b70c5db984a/.clang-format#L6-L7